examples/integration_with_error.py

```diff
@@ -42,51 +44,38 @@
 data /= 0.5 * 0.25  # the dt in the integral for both dims
 data /= sqrt(ndshape(data)["ph1"] * ndshape(data)["ph2"])  # normalization
 # }}}
+data.ft("t2", shift=True)
+dt = data.get_ft_prop("t2","dt")
+# {{{ vector-normalize the FT
+data /= sqrt(ndshape(data)['t2']) * dt
+# }}}
 # {{{ First, run the code that automatically chooses integration bounds
 # and also assigns error
 fl.next("compare manual vs. automatic", legend=True)
-error_pathway = (
-    set(((j, k) for j in range(2) for k in range(4)))
-    - set(excluded_pathways)
-    - set([(signal_pathway["ph1"], signal_pathway["ph2"])])
+error_pathway = (set(((j, k) for j in range(ndshape(data)['ph1']) for k in range(ndshape(data)['ph2'])))
+        - set(excluded_pathways)
+        - set([(signal_pathway['ph1'], signal_pathway['ph2'])])
 )
-error_pathway = [{"ph1": j, "ph2": k} for j, k in error_pathway]
-data = integral_w_errors(data, signal_pathway, error_pathway, fl=fl)
-fl.plot(data, ".", label="auto: real", capsize=6)
-fl.plot(data.imag, ".", label="auto: imaginary", capsize=6)
+error_pathway = [{'ph1':j, 'ph2':k} for j,k in error_pathway]
+s_int, returned_frq_slice = integral_w_errors(data, signal_pathway, error_pathway,
+        fl=fl, return_frq_slice=True)
+fl.plot(s_int, ".", label="fully auto: real", capsize=6)
+fl.plot(s_int.imag, ".", label="fully auto: imaginary", capsize=6)
 # }}}
 logger.debug(strm("check the std after FT", std(data["ph1", 0]["ph2", 0].data.real)))
 # the sqrt on the next line accounts for the var(real)+var(imag)
-std_off_pathway = data["ph1", 0]["ph2", 0].C.run(lambda x: abs(x) ** 2).mean_all_but(
-    "t2"
-).mean("t2").run(sqrt) / sqrt(2)
-logger.debug(strm("check the std after FT -- manually", std_off_pathway))
-logger.debug(
-    strm(
-        "check the std after FT -- manually don't collapse vd",
-        data["ph1", 0]["ph2", 0]
-        .C.run(lambda x: abs(x) ** 2)
-        .mean_all_but(["t2", "vd"])
-        .mean("t2")
-        .run(sqrt)
-        / sqrt(2),
-    )
-)
-dt = data.get_ft_prop("t2","dt")
-data.ft("t2", shift=True)
-# {{{ vector-normalize the FT
-data /= sqrt(ndshape(data)["t2"]) * dt
-# }}}
 fl.next("raw data")
 fl.image(data, alpha=0.5)
 fl.next("real part of raw data")
 fl.image(data.real, alpha=0.5)
 fl.next("compare manual vs. automatic")
 # here I plot w/ manually chosen integration bounds:
-for bounds in [
-    manual_slice,  # leave this as a loop so user can experiment with different bounds
+for bounds,thislabel in [
+    (manual_slice,'manual bounds'),  # leave this as a loop so user can experiment with different bounds
+    (tuple(returned_frq_slice),'auto bounds'),  # leave this as a loop so user can experiment with different bounds
 ]:
```

Comment:
the point here is that we want to see how we did with our choice of bounds.  As noted  in #44 , we did poorly